src/ngx_http_lua_shdict.c

```diff
@@ -106,32 +103,6 @@ ngx_http_lua_shdict_init_zone(ngx_shm_zone_t *shm_zone, void *data)
     ctx->shpool->log_nomem = 0;
 #endif
 
-done:
-
-    dd("get lmcf");
-
-    lmcf = ctx->main_conf;
-
-    dd("lmcf->lua: %p", lmcf->lua);
-
-    lmcf->shm_zones_inited++;
-
-    if (lmcf->shm_zones_inited == lmcf->shm_zones->nelts
-        && lmcf->init_handler)
-    {
-        saved_cycle = ngx_cycle;
-        ngx_cycle = ctx->cycle;
```

Comment:
@agentzh I moved this to `ngx_http_lua_shared_memary_init`, and that is exec after `user_init_func`. What do you think?

Thanks!